packages/ibm-products/src/components/Tearsheet/TearsheetNarrow.tsx

```diff
@@ -67,7 +63,7 @@ interface TearsheetNarrowProps extends PropsWithChildren {
    * **Note:** This prop is only required if a close icon is shown, i.e. if
    * there are a no navigation actions and/or hasCloseIcon is true.
    */
-  closeIconDescription?: closeIconDescriptionTypes;
+  closeIconDescription?: string;
```

Comment:
Do we still need this if we include `CloseIconDescriptionTypes` in like 122-124?

In that article @devadula-nandan recently shared[^1], it seemed like we might not need to?

[^1]: https://dev.to/maissenayed/conditional-react-props-with-typescript-43lg